src/commands/support/reportedissues.ts

```diff
@@ -7,12 +7,32 @@ import Logger from '../../lib/logger';
 
 const FBW_DOCS_REPORTED_ISSUES_URL = 'https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/';
 const FBW_DOCS_AUTOPILOT_ISSUES_URL = 'https://docs.flybywiresim.com/fbw-a32nx/feature-guides/autopilot-fbw/#typical-issues-and-how-to-solve-them';
+const FBW_DOCS_SIMBRIDGE_ISSUES_URL = 'https://docs.flybywiresim.com/simbridge/troubleshooting/';
 
 const genericReportedIssuesEmbed = makeEmbed({
     title: 'FlyByWire A32NX | Reported Issues',
     description: `Please see [this link](${FBW_DOCS_REPORTED_ISSUES_URL}) for a current list of reported issues.`,
 });
 
+const helpEmbed = makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: 'The Command lists one or up to 4 links to sections in the reported issues page of the documentation according to the search term specific by the following words. If no search term is given or no fitting section is found, a general link is shown. \n Search terms can be terminated by one of the following symbols or a line break: . - >  \n With the list subcommand you can see a list of available subcommands.',
```

Comment:
```suggestion
    description: 'The Command lists one or up to 4 links to sections in the reported issues page of the documentation according to the search term specific by the following words. If no search term is given or no fitting section is found, a general link is shown. \n Search terms can be terminated by one of the following symbols or a line break: . - > / \ \n With the list subcommand you can see a list of available subcommands.',
```
Adding the missing characters.